src/tile.js

```diff
@@ -45,8 +45,8 @@
  * @param {Boolean} exists Is this tile a part of a sparse image? ( Also has
  *      this tile failed to load? )
  * @param {String|Function} url The URL of this tile's image or a function that returns a url.
- * @param {CanvasRenderingContext2D} context2D The context2D of this tile if it
- *      is provided directly by the tile source.
+ * @param {?CanvasRenderingContext2D} context2D The context2D of this tile if it
```

Comment:
What's the `?` for? I'm not familiar with that... Is it just saying it's optional? If so, we use the `[context2D]` format more commonly here.